presentation/testing/src/test/IModelUtilities.test.ts

```diff
@@ -0,0 +1,174 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { buildTestIModel, IModelBuilder } from "../presentation-testing";
+import * as moq from "typemoq";
+import { CodeSpecs, IModelDb, IModelJsFs, SnapshotDb } from "@itwin/core-backend";
+import { BisCodeSpec, CodeScopeProps, CodeSpec, ElementAspectProps, ElementProps, ModelProps } from "@itwin/core-common";
+import { configureForPromiseResult } from "@itwin/presentation-common/lib/cjs/test";
+import { SnapshotConnection } from "@itwin/core-frontend";
+import sinon from "sinon";
+import { expect } from "chai";
+
+describe.only("IModelUtilities", () => {
+  let imodelMock: moq.IMock<IModelDb>;
+  let modelsMock: moq.IMock<IModelDb.Models>;
+  let elementsMock: moq.IMock<IModelDb.Elements>;
+  let codeSpecsMock: moq.IMock<CodeSpecs>;
+  let codeSpecMock: moq.IMock<CodeSpec>;
+
+  let modelPropsMock: moq.IMock<ModelProps>;
+  let elementPropsMock: moq.IMock<ElementProps>;
+  let elementAspectPropsMock: moq.IMock<ElementAspectProps>;
+  let codeScopePropsMock: moq.IMock<CodeScopeProps>;
+
+  let snapshotDbMock: moq.IMock<SnapshotDb>;
+  let snapshotConnectionMock: moq.IMock<SnapshotConnection>;
```

Comment:
it looks like every test creates and sets up these mocks individually, so there's no point in keeping them at this scope - they can be declared and defined in individual tests

Reply:
Done